controllers/contacts.js

```diff
@@ -0,0 +1,57 @@
+const { Contact } = require('../models/contact');
+
+const { HttpError, ctrlWrapper } = require('../helpers');
+
+const getAll = async (req, res) => {
```

Comment:
```suggestion
const getAllContacts = async (req, res) => {
```
назви інших контроллерів також поправте